src/js/components/ThumbsRating/index.d.ts

```diff
@@ -0,0 +1,24 @@
+import * as React from 'react';
+import { RadioButtonGroupProps } from '../RadioButtonGroup/index';
+export interface ThumbsRating {
+  color?: string | object;
+  options: (
```

Comment:
```suggestion
  options?: (
```
Should this be like this?